Services/DataX.Config/DataX.Config/ConfigGeneration/Processor/S500_ResolveOutputs.cs

```diff
@@ -81,6 +81,7 @@ public override async Task<FlowGuiConfig> HandleSensitiveData(FlowGuiConfig guiC
                             keyvaultName: RuntimeKeyVaultName.Value,
                             secretName: secretName,
                             secretValue: connStr,
+                            uriPrefix: (Configuration[Constants.ConfigSettingName_SparkType].Length > 0 && Configuration[Constants.ConfigSettingName_SparkType] == Constants.SparkTypeDataBricks) ? Constants.PrefixSecretScope : Constants.PrefixKeyVault,
```

Comment:
Seems like a common pattern, can this be extracted/refactored?

Reply:
Done. Refactored it in KeyVaultClient